tests/testthat/test-summarise_draws.R

```diff
@@ -82,12 +82,12 @@ test_that(paste(
   x <- as_draws_array(test_array)
   sum_x <- summarise_draws(x)
   parsum_x <- summarise_draws(x, .cores = cores)
-  expect_identical(sum_x, parsum_x)
```

Comment:
I had to change this test and the following one to expect_equal as I was getting "equal but not identical" errors for them on my machine. Not sure why, but I assume this is related to changes to matrixStats impacting parallel execution in some way? In any case I don't think we need to worry if the parallel version produces identical results as long as it is equal down to the default low tolerance?

Reply:
I agree. `expect_identical` is likely too strict in many cases.

Reply:
Great!